### PR TITLE
AB explainer snippet

### PR DESCRIPTION
**Pull request by regiskuckaertz**
An AB test to measure the effectiveness of the "snippet" concept we came up with during our design sprint a few weeks ago.

todo:
- [x] set correct test settings
- [x] get explainer IDs
- ~~get destination pages URLs~~

@gustavpursche Explainers are currently interactive atoms but we're working to turn them into first-class citizens in `frontend`. This is a first step in doing, any feedback welcome dude 👼

### CODE REVIEW

File: common/app/views/support/HtmlCleaner.scala [hunk 1]
```diff
@@ -785,3 +785,25 @@ case class CommercialComponentHigh(isPaidContent: Boolean, isNetworkFront: Boole
   }
 
 }
+
+object ExplainerCleaner extends HtmlCleaner {
```
Comment (annebyrne) [inline common/app/views/support/HtmlCleaner.scala]: Will this cleaner be used for both explainer-snippets and explainers?

**Comment by annebyrne** [comment]
Other than comment about semantics LGTM :+1:

File: common/app/views/support/HtmlCleaner.scala (hunk 1 above)
Comment (regiskuckaertz) [inline common/app/views/support/HtmlCleaner.scala]: Yes, as long as they are interactives. That's why I need the specific IDs of those I want to replace.

File: static/src/stylesheets/module/atoms/_explainers.scss [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+    path {
+        fill: currentColor;
+    }
+}
+
+/** THEMING */
+
+.explainer-snippet--light {
+    background: #f1f1f1;
```
Comment (gustavpursche) [inline static/src/stylesheets/module/atoms/_explainers.scss]: `$neutral-6` is very close. Could you please use one of the pre-defined variables instead?

Comment (benwuersching) [inline static/src/stylesheets/module/atoms/_explainers.scss]: #f1f1f1 is part of the colour palette for our Analysis tone. It's used as bg colour for the H1 (e.g. https://www.theguardian.com/world/2017/may/07/emmanuel-macron-elected-french-president-what-happens-now) and as bg colour for Analysis-tone rich links (e.g. bottom left column on https://www.theguardian.com/world/2017/may/17/macron-austrian-minister-sebastian-kurz-french-election).

Comment (gustavpursche) [inline static/src/stylesheets/module/atoms/_explainers.scss]: Ah, oh! So just ignore my comment :)

**Pull request merged**